core/src/main/java/com/cryptape/cita/utils/TransactionUtil.java

```diff
@@ -104,7 +104,7 @@ public static boolean verifySignature(String addr, String content)
         long quota = blockChainTx.getQuota();
         long validUntilBlock = blockChainTx.getValidUntilBlock();
         String data = bytesToHexString(blockChainTx.getData());
-        String value = bytesToHexString(blockChainTx.getValue());
+        String value = "0x"+bytesToHexString(blockChainTx.getValue()); // the param value of Transaction class need add format tag
```

Comment:
Please don't add prefix `0x` directly. If the value contains `0x`, it will cause mistake. You can find a util method from utils module.

Reply:
thanks fixed